app/build.gradle

```diff
@@ -31,15 +18,10 @@ def loadExtraProperties(String fileName) {
 
 loadExtraProperties("treetracker.keys.properties")
 
-
 android {
     compileSdkVersion 29
     buildToolsVersion '29.0.0'
-    playAccountConfigs {
-        defaultAccountConfig {
-            jsonFile = file('keys.json')
```

Comment:
I forgot what this was meant for.

Reply:
This is for the play console plugin I think.